index.html

```diff
@@ -214,10 +214,21 @@ <h2 id="about__contactUs">CONTACT US</h2>
 					</p>
 					<form action="https://formspree.io/f/mbjqvypj" method="post">
 						<div class="input">
-							<input type="text" name="name" placeholder="Text Input" maxlength="30" required/>
+							<input
+								type="text"
+								name="name"
+								placeholder="Text Input"
+								maxlength="30"
+								required
+							/>
 						</div>
 						<div class="input">
-							<input type="email" name="email" placeholder="jeremy@gmail.com" required/>
+							<input
+								type="email"
+								name="email"
+								placeholder="jeremy@gmail.com"
```

Comment:
- [ ] Kindly stick to the figma design
## Your design
<img width="364" alt="Screen Shot 2022-08-22 at 08 47 11" src="https://user-images.githubusercontent.com/28307960/185856640-d585623d-fb17-46cf-b219-345026a6999f.png">

## Figma design
<img width="470" alt="Screen Shot 2022-08-22 at 08 46 58" src="https://user-images.githubusercontent.com/28307960/185856716-6995c1f9-143d-42a5-9cc5-ada860f9c1b1.png">